Big-Spender/readme.md

```diff
@@ -125,7 +144,10 @@ CREATE YOUR QUERY HERE
 
 ```sql
 INSERT YOUR QUERIES HERE
-
+insert into spends (amount, description, supplier_inv_no, transaction_no, date) values (32000,'Computer Hardware Dell', 3780119655, 38104091, '2021-08-19');
+insert into suppliers (supplier) values('Dell');
+insert into expense_areas (expense_area ) values ('IT');
+update spends set expense_area_id = 46, supplier_id = 66 where id = 346;
```

Comment:
I like the use of update here! This is a clever way of doing it. How else could you insert the tables so you wouldn't need to do an update? (not a problem, just a thought!)

Reply:
insert into suppliers (supplier) values('Dell');
insert into expense_areas (expense_area ) values ('IT');
insert into spends (amount, description, supplier_inv_no, transaction_no, date, expense_area_id,  supplier_id) values (32000, 'Computer Hardware Dell', 3780119655, 38104091, '2021-08-19', 46, 66);

or

insert into suppliers (supplier) values('Dell');
insert into expense_areas (expense_area ) values ('IT');
insert into spends (amount, description, supplier_inv_no, transaction_no, date) values (32000, 'Computer Hardware Dell', 3780119655, 38104091, '2021-08-19');